plottable.css

```diff
@@ -44,6 +44,15 @@ svg.plottable {
   pointer-events: visibleFill;
 }
 
+/* Root-level html components are relative, all others are absolute */
+div.plottable {
```

Comment:
you shouldn't qualify class selectors with an element tag; it's not performant. I think you should use a different class to do this styling adjustment instead.

https://github.com/stylelint/stylelint/tree/master/lib/rules/selector-no-qualifying-type